bin/ssh_switch

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/expect -f
+spawn ssh manager@192.168.1.1
```

Comment:
filename doesn't seem right.... I would expect ssh_switch to open a ssh connection to the switch, rather it's resetting factory default... suggest a better name!